CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 💪 1.10.2
+
+- ### Fixes
+
+  - **Fixes bug preventing `wrangler publish` from deleting over 10000 keys - [ashleymichal], [issue/1398] [pull/1400]**
+
+    In 1.10.0 we removed the limit on bulk KV PUT and DELETE operations by batching the values but missed batched deletes for Workers Sites. Now with some refactoring all batched operations use the same logic and have the same behavior.
```

Comment:
little thing, but maybe we could link to 1.10.0/the specific change here? i have kind of a vague sense of what "bulk KV operations" mean but might be nice to improve the paper trail here

Reply:
oh, saw this just got merged! feel free to ignore this

Reply:
Yeah sorry, I totally missed this but that's a good idea. Next release I'll add a suggestion to link the PR.